BackendNew/models/database.js

```diff
@@ -0,0 +1,16 @@
+const mysql = require("mysql");
+const dbConfig = require("../config/database.js");
+
+var connection = mysql.createConnection({
+  host: dbConfig.HOST,
+  user: dbConfig.USER,
+  password: dbConfig.PASSWORD,
+  database: dbConfig.DB
+});
+
+connection.connect(function(err) {
+  if (err) throw err;
+  console.log("Connected!");
+});
```

Comment:
can you please add some code so that it will automatically reconnect in case if DB connection is lost somehow

Reply:
Okay

Reply:
> can you please add some code so that it will automatically reconnect in case if DB connection is lost somehow

@hasinichathu I have fixed that issue